tauri/src/client/utils/url-highlighter.ts

```
@@ -0,0 +1,224 @@
+    for (const textNode of textNodes) {
+      const nodeText = textNode.textContent || '';
+      const nodeStart = currentPos;
+      const nodeEnd = currentPos + nodeText.length;
+
+      // Check if this text node contains part of our link
+      if (!foundEnd && nodeEnd > startCol && nodeStart < endCol) {
+        const linkStart = Math.max(0, startCol - nodeStart);
+        const linkEnd = Math.min(nodeText.length, endCol - nodeStart);
+
+        if (linkStart < linkEnd) {
+          this.wrapTextInClickable(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)